alpha/alpha-fsm-channel-kafka/src/main/java/org/apache/servicecomb/pack/alpha/fsm/channel/kafka/KafkaChannelAutoConfiguration.java

```diff
@@ -0,0 +1,125 @@
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Lazy;
+import org.springframework.kafka.config.ConcurrentKafkaListenerContainerFactory;
+import org.springframework.kafka.config.KafkaListenerContainerFactory;
+import org.springframework.kafka.core.*;
+import org.springframework.kafka.listener.ConcurrentMessageListenerContainer;
+import org.springframework.kafka.support.serializer.JsonDeserializer;
+import org.springframework.kafka.support.serializer.JsonSerializer;
+
+import java.util.Map;
+
+@Configuration
+@ConditionalOnClass(KafkaProperties.class)
+@ConditionalOnProperty(value = "alpha.feature.akka.channel.type", havingValue = "kafka")
+public class KafkaChannelAutoConfiguration {
+
+    private static final Logger logger = LoggerFactory.getLogger(KafkaChannelAutoConfiguration.class);
+
+    @Value("${alpha.feature.akka.channel.kafka.topic:servicecomb-pack-actor-event}")
+    private String topic;
+
+    @Value("${spring.kafka.bootstrap-servers}")
+    private String bootstrap_servers;
+
+    @Value("${spring.kafka.consumer.group-id:servicecomb-pack}")
+    private String groupId;
+
+    @Value("${spring.kafka.consumer.properties.spring.json.trusted.packages:org.apache.servicecomb.pack.alpha.core.fsm.event,org.apache.servicecomb.pack.alpha.core.fsm.event.base,}org.apache.servicecomb.pack.alpha.core.fsm.event.internal")
+    private String trusted_packages;
+
+    @Bean
+    @ConditionalOnMissingBean
+    public ProducerFactory<String, Object> producerFactory(){
+        Map<String, Object> map = Maps.newHashMap();
+        map.put(ProducerConfig.BOOTSTRAP_SERVERS_CONFIG, bootstrap_servers);
+        map.put(ProducerConfig.KEY_SERIALIZER_CLASS_CONFIG, StringSerializer.class);
+        map.put(ProducerConfig.VALUE_SERIALIZER_CLASS_CONFIG, JsonSerializer.class);
+        map.put(ProducerConfig.RETRIES_CONFIG, 0);
+        map.put(ProducerConfig.BATCH_SIZE_CONFIG, 16304);
+        map.put(ProducerConfig.BUFFER_MEMORY_CONFIG, 33354432);
+
+        return new DefaultKafkaProducerFactory<>(map);
+    }
+
+    @Bean
+    @ConditionalOnMissingBean
+    public KafkaTemplate<String, Object> kafkaTemplate(){
+        return new KafkaTemplate<>(producerFactory());
+    }
+
+    @Bean
+    @ConditionalOnMissingBean
+    public ConsumerFactory<String, Object> consumerFactory(){
+        Map<String, Object> map = Maps.newHashMap();
+
+        map.put(ConsumerConfig.BOOTSTRAP_SERVERS_CONFIG, bootstrap_servers);
+        map.put(ConsumerConfig.GROUP_ID_CONFIG, groupId);
+        map.put(ConsumerConfig.KEY_DESERIALIZER_CLASS_CONFIG, StringDeserializer.class);
+        map.put(ConsumerConfig.VALUE_DESERIALIZER_CLASS_CONFIG, JsonDeserializer.class);
+        map.put(ConsumerConfig.AUTO_OFFSET_RESET_CONFIG, "earliest");
+        map.put(ConsumerConfig.ENABLE_AUTO_COMMIT_CONFIG, true);
+        map.put(ConsumerConfig.AUTO_COMMIT_INTERVAL_MS_CONFIG, 100);
```

Comment:
I suggest that some key parameters are best allowed to be modified by the user. 
e.g.
AUTO_OFFSET_RESET_CONFIG
AUTO_COMMIT_INTERVAL_MS_CONFIG

Reply:
Open parameters are configured for the user